colors/material.vim

```diff
@@ -165,21 +165,21 @@ call s:SetHighlight('DiffChange', s:bg, s:yellow, '')
 call s:SetHighlight('DiffText', s:bg, s:yellow, '')
 call s:SetHighlight('ErrorMsg', s:bg, s:red, 'bold')
 call s:SetHighlight('FoldColumn', s:line_numbers, s:bg, '')
-call s:SetHighlight('Folded', s:line_numbers, s:bg, '')
+call s:SetHighlight('Folded', s:comments, s:bg, 'bold')
 call s:SetHighlight('LineNr', s:line_numbers, '', '')
 call s:SetHighlight('MatchParen', s:comments, s:cyan, 'bold')
 call s:SetHighlight('ModeMsg', s:green, '', '')
 call s:SetHighlight('MoreMsg', s:green, '', '')
 call s:SetHighlight('NonText', s:comments, '', '')
 call s:SetHighlight('Normal', s:fg, s:bg, '')
-call s:SetHighlight('Pmenu', s:fg, s:bg, '')
+call s:SetHighlight('Pmenu', s:fg, s:line_highlight, '')
```

Comment:
This is looking a little harsh to me with the black. I like how it is now, but if you feel like it should be differentiated from the background color I also wouldn't be opposed to changing this to
```vim
call s:SetHighlight('Pmenu', s:fg, s:selection, '')
```
<img width="656" alt="Screen Shot 2019-07-02 at 2 49 47 PM" src="https://user-images.githubusercontent.com/7041728/60538536-d01de580-9cd8-11e9-8a7d-448c813271ed.png">

Reply:
Done in 22607db